### PR TITLE
Include dottyLib when custom -classpath is added

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -518,8 +518,8 @@ object Build {
         val args: Seq[String] = spaceDelimited("<arg>").parsed
 
         val fullArgs = args.span(_ != "-classpath") match {
-          case (beforeCp, Nil) => beforeCp ++ ("-classpath" :: dottyLib :: Nil)
-          case (beforeCp, rest) => beforeCp ++ rest
+          case (beforeCp, "-classpath" :: cp :: rest) => beforeCp ++ List("-classpath", cp + ":" + dottyLib) ++ rest
+          case (beforeCp, _) => beforeCp ++ ("-classpath" :: dottyLib :: Nil)
```
Comment (allanrenucci) [inline project/Build.scala]: Let's be consistent and use the `List(..)` syntax here too

File: project/Build.scala [hunk 2]
```diff
@@ -533,8 +533,8 @@ object Build {
...
```
Comment (allanrenucci) [inline project/Build.scala]: same

File: project/Build.scala (hunk 1 above)
Comment (nicolasstucki) [reply project/Build.scala]: Done

File: project/Build.scala (hunk 2 above)
Comment (nicolasstucki) [reply project/Build.scala]: Done

**Pull request merged**